### DISCUSSION TITLE
Looking for any CAN FD sample capture

### DISCUSSION THREAD

**Discussion by Matheus-Garbelini** [jgressmann/supercan#7] [General]
Hello, anyone would like to share a CAN FD sample capture of a real car?
It seems it's very hard to come across real data bases for CAN FD on the web.
Thanks.

**Comment by marckleinebudde** [comment]
Hey @Matheus-Garbelini, you may also ask on the Linux CAN mailing list: http://vger.kernel.org/vger-lists.html#linux-can

**Reply by Matheus-Garbelini** [reply]
Thanks I'll try asking there. It seems a bit funny and unclear why we have all the CAN FD infrastructure and hardware available, but not a single capture is available on the web for our own testing and understanding.
Regards.

**Reply by Matheus-Garbelini** [reply]
@marckleinebudde perhaps, the author of the repository @jgressmann  has sample CAN FD captures since he developed supercan?

**Reply by jgressmann** [reply]
@Matheus-Garbelini, I am sorry, I cannot provide data from an actual car. I typically use synthetic CAN frames for device / firmware / driver tests. 

There are numerous sites on the internet which provide information on how to 'hack' into the various vehicle CAN busses. You might be able to find the information you are looking for there.

**Comment by Matheus-Garbelini** [comment]
I see thanks. It seems KIA Sorento MQ4 and Hyundai Tucson 2021 are using CAN FD.
Will try to contact repair shops to check if someone has collected CAN FD data already.
Regards.
